copter/source/docs/systemid-mode-operation.rst

```diff
@@ -95,3 +95,101 @@ main loop frequency divided by a sub-sample factor.
+It is probable that there is a systemic cause for this (undesirable) response of the closed-loop system, which could be a nonlinearity in the system or a high-pass filter that is applied to the gyro measurements and therefore attentuates pitch rates of low frequencies?
+
+.. image:: ../images/BodeplotPitchClosedLoop.png
+:name: fig-freq-resp-axis8
+
+The next :ref:`figure<fig-io-signals-axis11>` shows the signals that were used for the system identification, consisting of the actual pitch rate as system output and the motor mixer input of the pitch axis (aka pitch rate pid output) as system input.
+It is again observable that the system is merely reacting to an excitation at low frequencies.
+Therefore, the supressing of amplitudes at low frequencies is probably not located in the PID controller, but in the system to be identified.
+
+.. image:: ../images/PitchRateAxis11IdentificationIOSignals.png
+:name: fig-io-signals-axis11
+
+The actual identification was carried out in MATLAB with the help of the System Identification Toolbox.
+The generation of the mentioned linear transfer function model was conducted in the frequency-domain, meaning that the frequency-response of the model was fitted to an estimated frequency-response of the system generated from the real-flight.
+Afterwards, the generated transfer function was integrated in a Simulink model together with the PID controller of the pitch axis.
+The simulation result is shown below.
+
+.. image:: ../images/PitchModelOutput.png
+:name: fig-comparison-model-realflight
+
+Comparing the system response measured in the real flight and the output of the identified pitch axis model in :ref:`figure<fig-io-signals-axis11>` illustrates that the system behaviour can only be consistenly approximated at higher frequencies.
+The differences between t=55s and t=80s are mainly caused by shifted phases, the amplitude is nearly the same.
+At lower frequencies, the model output actually follows the target pitch rate shown :ref:`above´fig-io-signals-axis8>`, while the system response measured during the test flight does not alter in large amplitudes and mainly consists of noise.
+It is important to note that the input and output signals had to be truncated in order to generate a model that leads to acceptable results.
+To be precise, only the data from 55s till the end was used for the identification.
+Therefore, the system response at frequencies that appeared ealier is the extrapolation of the frequency-response that was identified with the used data.
+This indicates that there is some component or effect in the system, that has to be considered in the identification process in order to generate a linear model that is capable to reproduce the complete behaviour of the real system.
```

Comment:
So this can be rewritten once you gather the data and redo your analysis.   To show how the model compares with flight test data, typically the bode plots of the frequency sweeps are compared.  Then the validation is shown through making a pulse input in the model axis and then showing a comparison of the flight test data to the simulation response with the pilot input fed into the simulation.  The comparison of the time domain data with a different excitation is a better way to show correlation.